### DISCUSSION TITLE
bug when changing skin layout

### DISCUSSION THREAD

**Discussion by smocer** [ppy/osu#15196] [Q&A] [answered]
Logs: [logs.zip](https://github.com/ppy/osu/files/7376458/logs.zip)
Used skin: [-            #(SK) Mathi 1.0 - NM.zip](https://github.com/ppy/osu/files/7376585/-.SK.Mathi.1.0.-.NM.zip)
Video with the bug: https://www.youtube.com/watch?v=ZRzA3MvuySc

Bug:
After editing HUD elements' layout and restarting map with ~ (tilda), everything disappears, and you can't do anything except exiting game with alt+f4 or entering edit mode again with ctrl+shift+s (but it doesnt change anything).

Steps to reproduce:
1. Choose a map
2. Begin playing
3. Pause
4. Press ctrl+shift+s
5. Make some changes to the layout
6. Press "Save"
7. Press esc
8. Restart the map

Expected result:
Map runs normally, you can keep playing it or return to the main menu.

Actual result:
Interface disappears, map objects don't appear, also it is not possible neither to enter pause menu nor to exit to the main menu.

**Comment by bdach** [comment, marked as answer]
same thing as https://github.com/ppy/osu/discussions/14884 - you likely have two or more skins with the same `skin.ini` file contents

**Reply by peppy** [reply]
Will be fixed by #15204.
